R/publishStorm.R

```diff
@@ -0,0 +1,20 @@
+#'@title Publish storm
+#'@description Adds all attributes to an existing template
+#'@param templateId id of the template to instantiate
+#'@param filename a shapefile to post
+#'@param ... extra params to pass to checkAuth
+#'@details This will create all children associated with a storm
+#' and aggregate them under the template.
+#'@return TRUE if it worked FALSE otherwise
+#'@examples
+#'\dontrun{
+#'publishStorm("CAQw7M1", system.file("extdata", "Sandy_CIDA.zip", 
```

Comment:
in the future, it would be nice to point out that an auth step is required before this. Not necessary, but...